.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettifier
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.2
+        with:
+          prettier_options: --write **/*.+(js|html|css|json|md|yaml) --config .prettierrc
```

Comment:
Nit: I think by default Prettier will by default [look at `.prettierrc`](https://prettier.io/docs/en/configuration.html) in this repo, so `--config .prettierrc` might not be neccessary?

Reply:
You're right, I just tested this. I took a look at [this](https://github.com/creyD/prettier_action/issues/41#issuecomment-787981694) and decided to add it.